ts/features/fci/components/__tests__/__snapshots__/QtspClauseListItem.test.tsx.snap

```diff
@@ -510,7 +517,7 @@ exports[`Test QtspClauseListItem component should render a QtspClauseListItem co
                               }
                             >
                               <View
-                                accessible={true}
+                                accessible={false}
```

Comment:
Couldn't be this problematic in term of a11y? Why we have this change in termo of a11y?

Reply:
@shadowsheep1 This is the result of the PR relative to the `eslint-plugin-react-native-a11y`:
* https://github.com/pagopa/io-app-design-system/pull/227

Some `Selection` components, like `AnimatedCheckbox` or `AnimatedRadio` can't live alone (just the box/circle with the tick, nothing else), but they should be included in other components (like `CheckboxLabel` or `ListItemCheckbox`). For this reason, I preferred to set `accessible={false}` on those atomic components.

Reply:
This snapshot could give us an indication of a wrong approach to building the component (for example, a custom, non-standard component that is not mapped to the DS).

Reply:
Okay then. So this will improve a11y.

Reply:
> This snapshot could give us an indication of a wrong approach to building the component (for example, a custom, non-standard component that is not mapped to the DS).

@dmnplb  Wait, are you telling this should be check by the component owners? Maybe we should tag them?

Reply:
@shadowsheep1 Just checked. The `SignatureFieldItem` correctly uses `ListItemCheckbox` (which manages the accessibility state at a parent level), so there's no accessibility regression. @LazyAfternoons Can you confirm?

Reply:
@hevelius @LazyAfternoons have a look if after the merge of this PR you should address something for a11y in those components.

Reply:
No regression should arise but we'll check, thank you.